src/snow_rs/lib/dask_utils.py

```diff
@@ -52,4 +53,5 @@ def run_with_client(cores, memory):
     try:
         yield client
     finally:
+        time.sleep(4)
```

Comment:
Curious about the errors you were seeing? Four seconds is quite a lot and slows things down quite a bit.

Reply:
Agreed, this probably isn't worth addressing, especially if you haven't had any issues. Maybe is was just a high load day. I can try to dig up the error message if you are curious.

Reply:
How about we remove this from this PR and log an issue for future reference?